gini/gini-vision-lib-ios
# Fixes image picker animation speed

### Description
When using old iOS devices, the image selection was really slow (there was a delay between the selection and the animation).

### How to test
Run the example app in an old device (iPad 3 Retina for instance), and check that the animation in the cells is fixed)

### Merging
Merge after #294
